src/commands/report-failure.test.ts

```diff
@@ -0,0 +1,68 @@
+import * as path from 'path';
+import { Polly } from '@pollyjs/core';
+import NodeHttpAdapter from '@pollyjs/adapter-node-http';
+import FSPersister from '@pollyjs/persister-fs';
+import reportFailure from './report-failure';
+import setupHardRejection from 'hard-rejection';
+
+class SanitizingPersister extends FSPersister {
+  static get id() {
+    return 'sanitizing-fs';
+  }
+
+  get options() {
+    return {
+      recordingsDir: path.resolve(__dirname, '__recordings__'),
+    };
+  }
+
+  // ensure that the authorization token is not written to disk
+  saveRecording(recordingId: string, data: any) {
```

Comment:
let's at least leave a `//TODO: add type for data`

Reply:
actually better yet let's write an interface for what we use if we can't get an actual type